installbuilder/datafiles/Base_DSC.data

```diff
@@ -302,35 +336,72 @@ chown -R ${{RUN_AS_USER}} /opt/microsoft/omsagent/plugin
 # If this is ever changed/removed, coordinate with nxOMSAuditdPlugin author/maintainer(s)
 chown -R ${{RUN_AS_USER}} /opt/microsoft/omsconfig/Scripts
 
+# pythonVersion check must be repeated for each section
+if  [ ! "$(python2 --version 2>&1 | grep 'not found')" ]; then
+  echo "Using python"
+  pythonVersion="python"
+else
+  echo "Using python3"
+  pythonVersion="python3"
+fi
+
 # Set up built-in resource modules for OMS DSC
-su - omsagent -c "/opt/microsoft/omsconfig/Scripts/InstallModule.py /opt/microsoft/omsconfig/module_packages/nx_1.0.zip 0"
-su - omsagent -c "/opt/microsoft/omsconfig/Scripts/InstallModule.py /opt/microsoft/omsconfig/module_packages/nxOMSPerfCounter_2.2.zip 0"
-su - omsagent -c "/opt/microsoft/omsconfig/Scripts/InstallModule.py /opt/microsoft/omsconfig/module_packages/nxOMSSyslog_2.5.zip 0"
```

Comment:
why was the version dropped by 1 ?

Reply:
It seems it was missed as a part of merge with master. will merge again.  This was added to master after these changes were committed.